src/styles/Lesson7.scss

```diff
@@ -4,5 +4,15 @@
 // =======================================
 
 .lesson7slides {
-  width: 55%;
+  width: 100%;
+  // height: 100%; /* Set both width and height to 'auto' to maintain aspect ratio */
+  // max-width: 100%; /* Limit the width to the container's width if needed */
+  // max-height: 100%; /* Limit the height to the container's height if needed */
+}
+
+@media screen and (min-width: 1920px) and (min-height: 1080px) {
```

Comment:
I don't think this is all that necessary. Not to mention, it still doesn't look right on my screen due to screen size differences.
<img width="1511" alt="image" src="https://github.com/uclaacm/parcel-pointers/assets/78634083/f6f456e6-a001-4b7f-8cfd-2555416ad998">
I think you should take a look at just changing the slideshow's [styling](https://github.com/uclaacm/parcel-pointers/blob/main/src/styles/SlideShow.scss) because it seems to look better without it (maybe shrink back down the width a tiny bit)
<img width="351" alt="image" src="https://github.com/uclaacm/parcel-pointers/assets/78634083/8ce41e09-30bb-4c0a-8065-ea20a6249636">
<img width="1505" alt="image" src="https://github.com/uclaacm/parcel-pointers/assets/78634083/43cf5434-90d9-4238-be6a-c569c9d6de23">